python/lsst/alert/packet/schemaRegistry.py

```diff
@@ -63,6 +63,7 @@ def register_schema(self, schema, version):
         """
         schema_id = self.calculate_id(schema)
         self._version_to_id[version] = schema_id
+        self._ids[schema_id] = schema_id
```

Comment:
Should `_ids` just be a list?  It seems weird to have a dictionary with identical keys and values.

Reply:
Swapped to a list